src/rpc/masternode.cpp

```diff
@@ -114,234 +114,250 @@ UniValue getpoolinfo(const JSONRPCRequest& request)
     return obj;
 }
 
+void masternode_list_help()
```

Comment:
Should also reuse this function in `masternodelist` function (line 815+) to ensure consistency imo.